core/subprojects/README.md

```diff
@@ -0,0 +1,19 @@
+# core/subprojects
+
+## What's this? What _is_ this?
+
+- These are subprojects accessible to the `core` meson project.
+
+## icu
+
+- Ideally we would use meson's [wrapdb](https://mesonbuild.com/Wrapdb-projects.html), and in fact the files here are sourced from that project.
+- However, it requires a much newer meson version to work properly: 0.55 at least.
+- Patches for icu (i.e. meson build files) are in [`packagefiles/icu`](./packagefiles/icu/)
+- The file `packagefiles/icu-minimal.zip` is generated from the files in [`packagefiles/icu`](./packagefiles/icu/)
+- Regenerate it with these commands:
```

Comment:
So, given we can build with meson 1.0+ on our ba-bionic64ta agent now (which is where we are limited on meson version), we should be able to rely on system-supplied ICU (v66) for Ubuntu and thus simplify the integration here, am I right?

If so, does that simplify the ICU integration any further than what we have now? Then this should be ready to merge!

Reply:
We can drop the zip file (again!) that's the simplification we can do now. We can't use upstream wrap db AND minimize size yet. But I hope to upstream changes to make the icu configurable so we can someday depend on the wrap

Reply:
That's a good step forward, really happy to not include the .zip!

Reply:
> That's a good step forward, really happy to not include the .zip!

You ok if I force push just to drop the last couple commits? That will remove the zip I re added.

Reply:
ok

Reply:
Nevermind. That's not going to work with the merges.  Anyway, I did a revert and also another merge